### PR TITLE
S01e04 schemes

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: code_schemes/s01e04_yes_no_amb.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
+{
+    "Version": "0.0.0.1",
+    "Name": "S01E04 Yes/No/Amb",
+    "SchemeID": "Scheme-5addd5a4",
+    "Codes": [{
```
Comment (as2388) [inline code_schemes/s01e04_yes_no_amb.json:5]:
```suggestion
    "Codes": [
        {
```

File: code_schemes/icraf_s01e04.json [hunk 2]
```diff
@@ -3,6 +3,110 @@
     "Name": "ICRAF s01e04",
     "Version": "0.0.0.1",
     "Codes": [{
+            "StringValue": "uproot_cut_the_eucalyptus_tree",
+            "DisplayText": "uproot cut the eucalyptus tree",
```
Comment (as2388) [inline code_schemes/icraf_s01e04.json]:
```suggestion
            "DisplayText": "uproot or cut the eucalyptus tree",
```
?

File: code_schemes/icraf_s01e04.json [hunk 3]
```diff
@@ -3,6 +3,110 @@
...
+            "StringValue": "good_in_draining_swampy_areas",
+            "DisplayText": "good in draining swampy areas",
+            "VisibleInCoda": true,
+            "NumericValue": 10,
+            "CodeID": "code-6021f21f",
+            "CodeType": "Normal"
+        },
+        {
+            "StringValue": "euclayputs_is_harmless",
+            "DisplayText": "euclayputs is harmless",
```
Comment (as2388) [inline code_schemes/icraf_s01e04.json]:
```suggestion
            "DisplayText": "eucalyptus is harmless",
```

File: code_schemes/icraf_s01e04.json [hunk 4]
```diff
@@ -3,6 +3,110 @@
...
+            "VisibleInCoda": true,
+            "NumericValue": 11,
+            "CodeID": "code-5b1c8ec3",
+            "CodeType": "Normal"
+        },
+        {
+            "StringValue": "others",
+            "DisplayText": "others",
```
Comment (as2388) [inline code_schemes/icraf_s01e04.json]: Think this was called "other" in the past

**Review by as2388 — APPROVED** [review]

File: code_schemes/icraf_s01e04.json (hunk 4 above)
Comment (IsaackMwenda) [reply code_schemes/icraf_s01e04.json]: Yes let me standardise this thanks!

File: code_schemes/icraf_s01e04.json (hunk 3 above)
Comment (IsaackMwenda) [reply code_schemes/icraf_s01e04.json]: Nice catch thanks!

File: code_schemes/s01e04_yes_no_amb.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/s01e04_yes_no_amb.json:5]: Updated thanks

File: code_schemes/icraf_s01e04.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/icraf_s01e04.json]: Updated thanks

**Pull request merged**